packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -131,34 +146,406 @@ distributionUrl=https\\://services.gradle.org/distributions/gradle-$gradleVersio
 String getGradleVersionForAndroidPlugin(Directory directory, Logger logger) {
   final File buildFile = directory.childFile('build.gradle');
   if (!buildFile.existsSync()) {
-    logger.printTrace("$buildFile doesn't exist, assuming Gradle version: $templateDefaultGradleVersion");
+    // TODO should this be a warning?
+    logger.printTrace(
```

Comment:
@christopherfujino Do you think this should be a warning or some other higher level error?

Reply:
sounds like this is fine as a printTrace